src/libs/PusherConnectionManager.js

```diff
@@ -21,11 +22,18 @@ function init() {
      *
      * @params {string} eventName
      */
-    Pusher.registerSocketEventCallback((eventName, data) => {
+    Pusher.registerSocketEventCallback((eventName, error) => {
         switch (eventName) {
             case 'error':
-                Log.info('[PusherConnectionManager] error event', false, {error: data});
-                Session.reauthenticatePusher();
+                if (error && error.type === 'PusherError' && error.data.code === 1006) {
+                    Log.hmmm('[PusherConnectionManager] Channels Error 1006', {error});
+                } else if (error && error.type === 'PusherError' && error.data.code === 4201) {
+                    Log.hmmm('[PusherConnectionManager] Pong reply not received', {error});
+                } else if (error && error.type === 'WebSocketError') {
+                    Log.hmmm('[PusherConnectionManager] WebSocketError', {error});
+                } else {
+                    Log.alert(`${CONST.ERROR.ENSURE_BUGBOT} [PusherConnectionManager] Unknown error event`, {error});
+                }
```

Comment:
I know this stuff is a little redundant, but I'm enumerating these and logging an alert so we can clearly document all the errors that can possibly happen.

Reply:
Great addition